### ISSUE TITLE
Give config.json and settings.json more descriptive names

### ISSUE THREAD

**Issue by klingtnet** [remie/YouTransfer#129]
I think it's confusing that you need both files. What's the difference between config and settings, anyway? My suggestion would be to use `config.json` exclusively.

**Comment by remie** [comment]
@klingtnet : fair enough. The reason they are here is to make a distinction between the config that you can edit through the UI (settings.json) and the application config (of which some of the properties are required on application start). So basically, the config.json file is a combination of application settings and default user settings, where the settings.json is used to store the user settings (incl. overrides of default values).

I didn't feel comfortable making the config.json editable from the UI because if something goes wrong it would block the application from starting properly (which is not desirable).

**Comment by klingtnet** [comment]
@remie Thank you for your fast and detailed reply. Even though I understand your motivation to seperate both files, I still think they should get a more declarative name, like `base_config.json` and `run_configs.json` (which are also not the best choice :smile:).

**Comment by remie** [comment]
@klingtnet : how about `appconfig.js` and `usersettings.js`? I'm not very keen on underscore :)

**Comment by remie** [comment]
or `default.json` as a replacement for `config.json`?

**Comment by klingtnet** [comment]
That should do the trick! With a short note about the usage/difference in the README.md it should be crystal clear for anyone.

**Comment by remie** [comment]
:+1: it shall be done!

**Comment by remie** [comment]
@klingtnet : I'm keeping it open to make sure that the files are actually renamed in the next version :)

**Comment by klingtnet** [comment]
While checking my open issues I stumbled across this one, which is now over two years old but I wouldn't close it before checking if the files were renamed.
Starting `YouTransfer` in a docker container showed that the config files in `/opt/youtransfer` are still called `config.json` and `settings.json`.
I am not sure if it makes much sense to rename those files anymore so it is up to you to close this issue.
